packages/lib-classifier/src/store/TutorialStore.spec.js

```diff
@@ -256,15 +278,33 @@ describe('Model > TutorialStore', function () {
         setTutorialStepSpy.restore()
       }).then(done, done)
     })
+
+    it('should call setSeenTime if the id parameter is defined', function (done) {
+      rootStore = RootStore.create({
+        tutorials: TutorialStore.create(),
+        workflows: WorkflowStore.create()
+      }, {
+        authClient: authClientStubWithoutUser, client: clientStub()
+      })
+      const setSeenTimeSpy = sinon.spy(rootStore.tutorials, 'setSeenTime')
```

Comment:
ls this function tested elsewhere? I think would it be better to run it, and test the outcome, than to stub it.

Reply:
Yeah, there's a separate describe block just for the behavior of `setSeenTime` in isolation. This is just testing to make sure `setActiveTutorial` is calling `setSeenTime` as expected.